client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -48,6 +51,8 @@ const StatsInsights = ( props ) => {
 			<StatsNavigation section="insights" />
 			<div>
 				<PostingActivity />
+				<SectionHeader label={ translate( 'All Time Views' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'views all-time' )` *ES Score: 10.07*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).